### DISCUSSION TITLE
Installing Postal on Windows Docker desctop

### DISCUSSION THREAD

**Discussion by AlexNikMak** [postalserver/postal#2822] [Help with using Postal]
Sorry if this question is dumb, but is it possible to install Postal using docker desktop on Windows server 2022 with Hyper-V(NO WSL)?

**Comment by willpower232** [comment]
technically yes, if you have acces to docker compose then you should be good but using docker on non-linux platforms adds a lot of maintenance overhead so you may end up with more issues than if you just ran a linux OS in your Hyper V.
